README.md

```diff
@@ -0,0 +1,60 @@
+# Membrane Multimedia Framework: MembraneElementSdl
+
+This package provides an SDL-based video player.
+
+It is part of [Membrane Multimedia Framework](https://membraneframework.org).
+
+## Installation
+
+The package can be installed by adding `membrane_element_sdl` to your list of dependencies in `mix.exs`:
+
+```elixir
+def deps do
+  [
+    {:membrane_element_sdl, "~> 0.1.0"}
+  ]
+end
+```
+
+The docs can be found at [HexDocs](https://hexdocs.pm/membrane_element_sdl).
+
+## Usage
+
+The pipeline below displays a sample h264 video from the net:
```

Comment:
You should mention:
1. list of needed deps
1. a fact that it uses cnode and needs `empd -d` started